libs/coin-modules/coin-evm/src/__tests__/coin-tester/scenarios/ethereum.ts

```diff
@@ -0,0 +1,263 @@
+  await provider.waitForTransaction(hash);
+};
+
+const defaultNanoApp = { firmware: "2.2.3" as const, version: "1.10.4" as const };
+
+export const scenarioEthereum: Scenario<EvmTransaction> = {
+  name: "Ledger Live Basic ETH Transactions",
+  setup: async () => {
+    const [{ transport, onSignerConfirmation }] = await Promise.all([
+      spawnSpeculos(`/${defaultNanoApp.firmware}/Ethereum/app_${defaultNanoApp.version}.elf`),
+      spawnAnvil("https://rpc.ankr.com/eth"),
+    ]);
+
+    const provider = new providers.StaticJsonRpcProvider("http://127.0.0.1:8545");
+    const signerContext: Parameters<typeof resolver>[0] = (deviceId, fn) => fn(new Eth(transport));
+
+    setCoinConfig(() => ({
+      info: {
+        status: {
+          type: "active",
+        },
+        gasTracker: {
+          type: "ledger",
+          explorerId: "eth",
+        },
+        node: {
+          type: "external",
+          uri: "http://127.0.0.1:8545",
+        },
+        explorer: {
+          type: "ledger",
+          explorerId: "eth",
+        },
+      },
+    }));
+
+    const currencyBridge = buildCurrencyBridge(signerContext);
+    const accountBridge = buildAccountBridge(signerContext);
+    const getAddress = resolver(signerContext);
+    const { address } = await getAddress("", {
+      path: "44'/60'/0'/0/0",
+      currency: getCryptoCurrencyById("ethereum"),
+      derivationMode: "",
+    });
+
+    const scenarioAccount = makeAccount(address, ethereum);
+
+    await setBlock();
+
+    // Binance account
+    await impersonnateAccount({
+      provider,
+      data: ERC20Interface.encodeFunctionData("transfer", [
+        address,
+        ethers.utils.parseUnits("100", USDC_ON_ETHEREUM.units[0].magnitude),
+      ]),
+      to: USDC_ON_ETHEREUM.contractAddress,
+      addressToImpersonnate: "0x47ac0Fb4F2D84898e4D9E7b4DaB3C24507a6D503", // binance
+    });
+
+    // Bored Ape
+    await impersonnateAccount({
+      provider,
+      data: ERC721Interface.encodeFunctionData("transferFrom", [
+        "0x440Bcc7a1CF465EAFaBaE301D1D7739cbFe09dDA",
+        address,
+        "3368",
+      ]),
+      to: "0xBC4CA0EdA7647A8aB7C2061c2E118A18a936f13D",
+      addressToImpersonnate: "0x440Bcc7a1CF465EAFaBaE301D1D7739cbFe09dDA",
+    });
+
+    // Clone X
+    await impersonnateAccount({
+      provider,
+      data: ERC1155Interface.encodeFunctionData("safeTransferFrom", [
+        "0xa3cd1123f4860C0cC512C775Ab6DB6A3E3d1B1Ee",
+        address,
+        "951",
+        2,
+        "0x",
+      ]),
+      to: "0x348fc118bcc65a92dc033a951af153d14d945312",
+      addressToImpersonnate: "0xa3cd1123f4860C0cC512C775Ab6DB6A3E3d1B1Ee",
+    });
+
+    await getLogs();
+
+    return {
+      currencyBridge,
+      accountBridge,
+      account: scenarioAccount,
+      onSignerConfirmation,
+    };
+  },
+  beforeAll: account => {
+    expect(account.nfts?.length).toBe(2);
+  },
```

Comment:
Missing account.balance = 10k eth + 1 tokenAccount for USDC with balance = 100 usdc